### PR TITLE
v80 beta uplifts

### PR DESCRIPTION
**Pull request by liuche**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/Config.kt [hunk 1]
```diff
@@ -35,6 +33,12 @@ enum class ReleaseChannel {
             else -> false
         }
 
+    val isRelease: Boolean
+        get() = when (this) {
+            FennecProduction -> true
```
Comment (pocmo) [inline app/src/main/java/org/mozilla/fenix/Config.kt:38]: `FennecProduction` is org.mozilla.fenix, which is "Nightly" now (all of that will be cleaned up in #12827). So I wonder if that is correct here? It's definitely confusing to have `isRelease` return `true` for a Nightly build? Is that on `master` too? 🤔

Comment (ekager) [reply app/src/main/java/org/mozilla/fenix/Config.kt:38]: I thought that was `FenixProduction`? 🤔 I'm confused now. `fennecProduction` looks like it's `org.mozilla.firefox`?? Is the readme outdated again 😭

Comment (liuche) [reply app/src/main/java/org/mozilla/fenix/Config.kt:38]: @pocmo could you confirm what I'm seeing? In [the build variants in build.gradle](https://github.com/mozilla-mobile/fenix/blob/master/app/build.gradle#L82) `fennecProduction` is "Firefox Release" (and `fenixProduction` is Nightly).

I'm not sure about where [this nightly/production naming](https://github.com/mozilla-mobile/fenix/blob/master/taskcluster/ci/push-apk/kind.yml#L30) comes from, but it also seems to confirm at least that `fennecProduction` is Firefox Release.

Comment (pocmo) [reply app/src/main/java/org/mozilla/fenix/Config.kt:38]: @ekager Oh, I'm totally sorry. You are right. I was confused. 🤦

**Pull request merged**